lldap_wrapper.py

```
@@ -0,0 +1,126 @@
+            )
+            if response.status_code not in {200, 204}:
+                return (
+                    False,
+                    f"Authelia reset failed: {response.status_code} - {response.text}",
+                )
+        except requests.RequestException as e:
+            return (
+                False,
+                f"Authelia reset failed (network error). {type(e).__name__}: {e}",
+            )
+        except Exception as e:  # noqa: BLE001
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)